packages/terra-clinical-header/src/Header.jsx

```diff
@@ -59,6 +61,11 @@ const propTypes = {
    * A Boolean indicating if element is a subheader.
    */
   isSubheader: PropTypes.bool,
+
+  /**
+   * Callback function triggered via hyperlink button title.
+   */
+  onTextClick: PropTypes.func,
```

Comment:
Shouldn't this be "onClick"?

Reply:
I think it will be clear.  We are passing to the onClick of the Hyperlink component.  It would be consistent.

Reply:
Sounds good, got that updated here and also added a bi more documentation for the prop https://github.com/cerner/terra-clinical/pull/923/commits/132963b4195a9ee2d4499cec60faa81ad3d879bf

Thank you!